File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L4/05/inputoutput.ts
import readline from 'readline';

function hitHeight(kaki: number, inc: number): number {
  return kaki * 12 + inc;
}

function hitBmi(bobot: number, height: number): number {
  return bobot * 0.45359237 / ((height * 0.0254) * (height * 0.0254));
}

async function main() {
  const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
  });

  console.log('Enter weight in pounds: ');
  const bobot = await new Promise<number>((resolve) => {
    rl.question('', (answer) => {
      resolve(parseFloat(answer));
    });
  });

  console.log('Enter feet: ');
  const kaki = await new Promise<number>((resolve) => {
    rl.question('', (answer) => {
      resolve(parseFloat(answer));
    });
  });

  console.log('Enter inches: ');
  const inc = await new Promise<number>((resolve) => {
    rl.question('', (answer) => {
      resolve(parseFloat(answer));
    });
  });

  const height = hitHeight(kaki, inc);
  const bmi = hitBmi(bobot, height);

  console.log(`BMI is ${bmi}`);
  if (bmi < 18.5) {
    console.log('Underweight');
  } else if (bmi < 25) {
    console.log('Normal');
  } else if (bmi < 30) {
    console.log('Overweight');
  } else {
    console.log('Obese');
  }

  rl.close();
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/plagiarized/L3/04/inputoutput.kt
package IR

import java.util.Scanner

fun main() {
    var kaki: Double
    var bobot: Double
    var bmi: Double
    var inc: Double
    var height: Double
    val input = Scanner(System.`in`)
    print("Enter weight in pounds: ")
    bobot = input.nextDouble()
    print("Enter feet: ")
    kaki = input.nextDouble()
    print("Enter inches: ")
    inc = input.nextDouble()
    height = kaki * 12 + inc
    bmi = bobot * 0.45359237 / ((height * 0.0254) * (height * 0.0254))

    // hasil
    println("BMI is $bmi")
    when {
        bmi < 18.5 -> println("Underweight")
        bmi < 25 -> println("Normal")
        bmi < 30 -> println("Overweight")
        else -> println("Obese")
    }
}

